core/src/bundle/usBundle.cpp

```diff
@@ -138,6 +138,11 @@ void Bundle::Stop(uint32_t options)
 
 void Bundle::Uninstall()
 {
+  if (!IsSharedLibrary(GetLocation()))
+  {
+    throw std::runtime_error("Bundles embedded in the executable are not uninstallable");
```

Comment:
The "not uninstallable" part sounds awkward to me. What do you think about something like

> Bundles embedded in an executable cannot be uninstalled.

?